.changeset/fast-walls-carry.md

```diff
@@ -0,0 +1,24 @@
+---
+'@backstage/app-defaults': patch
+'@backstage/core-app-api': patch
+'@backstage/core-plugin-api': patch
+'@backstage/plugin-catalog': patch
```

Comment:
It's the correct set of packages, but it's not really what changed in these packages. As a changelog reader the below description doesn't really make sense for the catalog plugin for example. It's best to split this up into multiple changesets that describes the changes towards each package